src/js/osweb/backends/keyboard.js

```diff
@@ -162,10 +162,14 @@ export default class Keyboard {
    * @return {Array} - List of default values for the given responses.
    */
   _get_default_from_synoniem (responses) {
-    const defaults = []
+    let defaults = []

```

Comment:
Defaults can remain a const as you are not reassigning it anywhere (the linter will complain about and/or fix this ;) )

Reply:
Thanks. (I assumed that it would be poor style to use `const` for something that does change, even if it's not re-assigned.)